_includes/common-scripts.html

```diff
@@ -0,0 +1,11 @@
+<script src="{{baseurl}}/assets/vendor/jquery-3.3.1.min.js"></script>
+<script src="{{baseurl}}/assets/vendor/bootstrap-4.1.3-dist/js/bootstrap.bundle.js"></script>
+
+<script>
```

Comment:
I wouldn't mix HTML and Js, specially in new code...

Reply:
mix? common-scripts.html only has the common javascript code for all pages

Reply:
But the generated website will have a mix of HTML and Javascript